ControllerDriver/networkhandler.cpp

```diff
@@ -0,0 +1,108 @@
+#include "networkhandler.h"
+#include "queue.h"
+#include <WiFi.h>
+#include "networkconfig.h"
+#include "main.h"
+
+const char* ssid     = NETSSID;
+const char* password = NETPASS;
+const char* host = NETHOST;
+
+const int serverPort = 11000;
+
+unsigned long waitUntil = 0;
+
+// Use WiFiClient class to create TCP connections
+WiFiClient client;
+
+int wStatus = WL_IDLE_STATUS;
+QueueArray<struct Packet*> buffer;
+
+void handleNetworkStuff() {
+  if(waitUntil < getLoopTime()) {
```

Comment:
Bit of a nitpick, but any reason you're not adding spaces after `if`s? I see a `while` block in this same file that has the spacing, so I'm not to sure why it's only missing on `if`s. Consistent style is probably something we should all work out together :)

Reply:
Pure habit. We should indeed define a style. I'll add the spaces.